mods/ra2/cursors.yaml

```diff
@@ -290,3 +292,6 @@ Cursors:
 			Length: 9
 		assaultmove-minimap:
 			Start: 9
+	mouse.shp: mouse
```

Comment:
Why did you create a new section for this but didn't add it to the first `mouse.shp` section?

Reply:
To keep the cursors next to each other.

Reply:
Can you please move it into the first section? On IRC we weren't quite sure if (ab)using the yaml merger in this way doesn't have any side effects. Keeping them separated will also be a potential bug source if anyone ever was to change the first section (not expecting it to be defined a second time).

Reply:
It would be pretty terrible if the yaml merger couldn't resolve this. There are unit tests in place https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Test/OpenRA.Game/MiniYamlTest.cs to check for problems.